multichain-testing/package.json

```
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "exit 0",
     "lint": "yarn tsc && yarn eslint .",
-    "lint-fix": "yarn lint:eslint --fix",
+    "lint-fix": "yarn lint --fix",
```

`yarn lint` just happens to end in eslint. Since we don't have `lint:` scripts in this package.json this one should call that directly.
```suggestion
    "lint-fix": "yarn eslint . --fix",
```